packages/firebase_auth/firebase_auth/test/firebase_auth_e2e.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2020, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:e2e/e2e.dart';
+
+void main() {
+  E2EWidgetsFlutterBinding.ensureInitialized();
+}
```

Comment:
Are there any tests that can be added here?

Reply:
I doubt if we can test any real sign in behavior? It's like the Google sign in plugin.